### PR TITLE
[GPSP] Prevent invalid messages from stopping the server

### PR DESCRIPTION
**Pull request by MikeIsAStar**

### CODE REVIEW

File: common/gamespy_message.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ type GameSpyCommand struct {
 }
 
 var (
-	InvalidGameSpyCommand = errors.New("invalid command received")
+	InvalidGameSpyCommand = errors.New("Invalid command received")
```
Comment (noahpistilli) [inline common/gamespy_message.go:16]: The Go Style guide states that the error string should not start with a capital letter.

Comment (mkwcat) [reply common/gamespy_message.go:16]: Oh, I didn't know that

Comment (mkwcat) [reply common/gamespy_message.go:16]: I've fixed it now

Comment (MikeIsAStar) [reply common/gamespy_message.go:16]: > The Go Style guide states that the error string should not start with a capital letter.

Thank you for catching that.

**Pull request merged**